public/js/humanize.js

```diff
@@ -8,6 +8,31 @@ var humanize = (function() {
     return Math.round(value * multiplier) / multiplier
   }
   return {
+    decimalParts: function(v, useCommas, precision) {
+      if (!precision || precision > 8) {
+        precision = 8
+      }
+      var formattedVal = parseFloat(v).toFixed(precision)
+      var chunks = formattedVal.split(".")
+      var int = useCommas ? parseInt(chunks[0]).toLocaleString() : chunks[0]
+      var decimal = (chunks[1] || "")
+      var i = decimal.length
+      var numTrailingZeros = 0
+      while (i--) {
+        if (decimal[i] == "0") {
+          numTrailingZeros++
+        } else {
+          break
+        }
+      }
+      var decimalVals = decimal.slice(0,decimal.length - numTrailingZeros)
+      var trailingZeros = (numTrailingZeros == 0) ? "" : decimal.slice(-(numTrailingZeros))
+      return "<span class='int'>"+int+"</span>"+
```

Comment:
When we build chunks of html strings that will get evaluated by the client, we should really be scrub out scripts as a security best practice. We could use [jQuery.parseHTML()](https://api.jquery.com/jquery.parsehtml/) for example.

Zepto doesn't come with that so we should either move to a lightweight build of jQuery ( since we're trying to keep page weight as low as possible ) or add some other template system in that guards against this efficiently.

Reply:
@chappjc if I strip out everything we don't need from the latest jQuery, its down to 24.72 Kb when minified and gzipped. You OK w/ switching to that? Zepto is currently 10.5Kb over the wire.